Framework/Algorithms/src/ConjoinXRuns.cpp

```diff
@@ -213,7 +211,7 @@ std::string ConjoinXRuns::checkLogEntry(MatrixWorkspace_sptr ws) const {
 
         // try if numeric time series, then the size must match to the
         // blocksize
-        const int blocksize = static_cast<int>(ws->blocksize());
+        const int blocksize = static_cast<int>(ws->y(0).size());
```

Comment:
Has this changed the behaviour in the case where a workspace has a different number of bins in each spectra? `blocksize` would throw in this case but `y(0)` will carry on?

Reply:
Good point, I forgot about the recent change to support variable-length histograms. So it should probably be kept as `blocksize()`, since we implementation does not support variable length (I think)?

Reply:
I don't think it would make sense in this case no.

Reply:
Nice catch! This method is called only from `validateInputs()` which would have checked the `blocksize()` on line 157 had I not mercilessly changed the call to `isCommonBins()`. Which is wrong. I'll revert the `isCommonBins()` change. We can then keep the `y(0).size()` call here.